src/renderer/components/NewProjectModal/RunTranscriptionView.tsx

```diff
@@ -1,47 +1,35 @@
-import { Box, styled, Typography, colors, Button } from '@mui/material';
+import { Box, styled, Stack, Typography, Button } from '@mui/material';
 import { useDispatch, useSelector } from 'react-redux';
+import colors from 'renderer/colors';
+import CloseIcon from '@mui/icons-material/Close';
+import IconButton from '@mui/material/IconButton';
```

Comment:
Can this default import be removed and replaced with the other named exports (Box, styled, Stack, Typography, Button). This will allow for tree-shaking if possible and is generally the best-practice syntax these days.